AGPs/Association-2020-Budget.md

```diff
@@ -0,0 +1,131 @@
+---
+AGP: N/A
+Title: Association 2020 Budget
+Author: Aragon Association
+Status: Stage III
+Track: Finance
+Created: 2019-11-19
+---
+
+# Association 2020 Budget
+
+## Address of the transfer recipient
+
+0x83eAA6D1449662f12eF87d64fF1b045318b27556
+
+## Amount of the transfer
+
+The association would like to request:
+- 1,500,000 DAI
+- 375,000 ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+The Aragon Association is the legal entity that oversees the Aragon Project's development during the decentralization process.
+
+The Association's responsibilities include: 
+- Spreading the values in the [Aragon Manifesto](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/).
+- Creating value for ANT holders.
+- Managing the [project's treasury](https://transparency.aragon.org).
+- Ensuring the long-term sustainability of the Aragon project.
+- Ensuring Aragon's adoption by overseeing development and awareness efforts.
+- Nurturing Aragon's governance and curating [governance proposals](https://aragon.org/project/governance).
+- Decentralizing all roles and responsibilities in order to transfer full governance power to ANT holders.
+
+In order to fulfill the above responsibilities for the Aragon Network, the Association needs capital to support it's team and cover it's various expenses for the Aragon Netw. 
```

Comment:
In order to fulfill the above responsibilities for the Aragon Network, the Association needs capital to support its team and cover Network related expenses. These include legal costs, HR, communications, accounting, banking, auditing, market making and more.

Reply:
(would replace with this)